webextension/js/popup.js

```diff
@@ -63,14 +64,20 @@ const Logic = {
   _panels: {},
 
   init() {
+    // Remove browserAction "upgraded" badge when opening panel
+    browser.browserAction.setBadgeBackgroundColor({color: ""});
```

Comment:
Can this be in it's own method like `clearVersionUpdate` and use state to store?

This would solve the other comment of only showing on update etc.